webpack.config.dev.js

```diff
@@ -6,7 +6,11 @@ const ExtractTextPlugin = require('extract-text-webpack-plugin');
 const path = require('path');
 const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
 
-const noVisualization = process.env.ANALYSE_PACK.toString() === 'false';
+if (process.env.ANALYSE_PACK) {
```

Comment:
ok this version doesn't  have the noVisualization at prods. my bad. About the var. We enforce es6. 
what you need to do is set a let as false(default) outside the if and check and set the value as true inside. you don't need the else.